test/binarySearch_test.js

```diff
@@ -0,0 +1,27 @@
+import { expect } from 'chai'
+import binarySearch from '../src/binarySearch.js'
+
+describe('binarySearch()', function() {
+  it('should be a function', function() {
+    expect(binarySearch).to.be.a('function')
+  })
+
+  it('should only take in an Array', function() {
+    expect(binarySearch({})).to.equal("Wrong Data Type")
+  })
+
+  it('should search for a number in an array and return its index', function() {
+    const numbers = [5, 10, 15, 20, 25, 30, 35, 40, 45, 50]
+    expect(binarySearch(numbers, 45)).to.equal(8)
+  })
+
+  it('should search for a number in an array and return its index', function() {
+    const numbers = [5, 10, 15]
+    expect(binarySearch(numbers, 5)).to.equal(0)
+  })
```

Comment:
This test is the same (and has the same description) as the test above it. Omit tests that test the exact same thing.